test/helpers/helpers.go

```diff
@@ -235,3 +220,41 @@ func CheckCatalogSourceConfigAndChildResourcesDeleted(client test.FrameworkClien
 	}
 	return nil
 }
+
+// CheckSuccessfulCreationCsc checks that a CatalogSourceConfig
+// and it's child resources were deployed.
+func CheckSuccessfulCreationCsc(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
```

Comment:
Going with the naming pattern you are following this should be `CheckCscSuccessfulCreation`